src/VoyagerHooksServiceProvider.php

```diff
@@ -13,20 +13,40 @@ class VoyagerHooksServiceProvider extends ServiceProvider
      */
     public function register()
     {
+        if (config('voyager-hooks.disable_in_production', false) && config('app.env', 'production') === 'production') {
+            return true;
+        }
+
+        if ($this->app->runningInConsole()) {
+            $config = dirname(__DIR__) . '/publishable/config/voyager-hooks.php';
+
+            $this->publishes(
+                [$config => config_path('voyager-hooks.php')],
+                'Voyager-hooks config'
+            );
+        }
+
+
         // Register the HooksServiceProvider
         $this->app->register(HooksServiceProvider::class);
 
         // Load views
-        $this->loadViewsFrom(__DIR__.'/../resources/views', 'voyager-hooks');
+        $this->loadViewsFrom(__DIR__ . '/../resources/views', 'voyager-hooks');
     }
 
     /**
      * Bootstrap the application services.
      *
      * @param \Illuminate\Events\Dispatcher $events
+     *
+     * @return bool
      */
     public function boot(Dispatcher $events)
     {
+        if (config('voyager-hooks.disable_in_production', false) && config('app.env', 'production') === 'production') {
+            return true;
```

Comment:
Is there a particular benefit to returning true? or did you just need `something` to return?  If the latter, you could just use `return;`

Reply:
It was simply to break it out of the boot method. I'll change it to return;

Reply:
Hey @JapSeyz.
Sorry for the delay.

Is there a reason that you would like to add `config('app.env', 'production') === 'production'` to the statement, rather than just having the configuration being a environment that could be set within the `.env` file?

However, it's a good idea to add the ability to disable this.

Reply:
I wanted to rely more on the built in app.env variable, as that's sure to always be specified.

Also in case someone forgets the feature or adding the env variable to the production machine, I don't want any irregularities to happen, but it seems to be a bit overkill for this situation. 

I'll change it up when I get home

Reply:
Honestly, I'm okay with checking against app.env. do you want it changed @marktopper, or just asking why it was done?

Reply:
With that statement in here, no one will be able to disable this for their development, staging or locale environment.
Only for their production environment.

Reply:
@marktopper I have updated the commits to make it more generic, and simply set a disable key, instead of disabled_in_production